conda/recipes/libcudf/meta.yaml

```
@@ -391,18 +435,38 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% else %}
         - {{ compiler('cuda') }}
+        {% endif %}
```

Do we want to limit this to CUDA 11 (like [was done in RMM]( https://github.com/rapidsai/rmm/blob/branch-23.08/conda/recipes/rmm/meta.yaml#L34-L37 ))?

```suggestion
      ignore_run_exports_from:
        {% if cuda_major == "11" %}
        - {{ compiler('cuda11') }}
        {% endif %}
```